src/app/pages/checkout-payment/payment-payone-ideal/payment-payone-ideal.component.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+} from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+import { Attribute } from 'ish-core/models/attribute/attribute.model';
+import { PaymentMethod } from 'ish-core/models/payment-method/payment-method.model';
+import { markAsDirtyRecursive } from 'ish-shared/forms/utils/form-utils';
+
+@Component({
+  selector: 'ish-payment-payone-ideal',
+  templateUrl: './payment-payone-ideal.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PaymentPayoneIdealComponent implements OnInit {
+  constructor(protected cd: ChangeDetectorRef) {}
+
+  /**
+   * payone payment method, needed to get configuration parameters
+   */
+  @Input() paymentMethod: PaymentMethod;
+
+  /**
+   * should be set to true by the parent, if component is visible
+   */
+  @Input() activated = false;
+
+  @Output() cancel = new EventEmitter<void>();
+  @Output() submit = new EventEmitter<{ parameters: Attribute<string>[]; saveAllowed: boolean }>();
+
+  payoneIDealForm = new FormGroup({});
+
+  bankGroupOptions: {
+    name: string;
+    value: string;
+  }[];
+
+  ngOnInit() {
+    this.payoneIDealForm.addControl('bankGroup', new FormControl('', [Validators.required]));
```

Comment:
use formly to generate the form using ish-field-select, you will find examples in the existing code, see also https://github.com/intershop/intershop-pwa/blob/develop/docs/guides/formly.md